source/rest-v2.md

```diff
@@ -667,6 +684,268 @@ The JSON response returns a list of collections with the following attributes:
+| source_url           | String   | Yes      | URL where the PDF file resides. HTTP and HTTPS are accepted, and it needs to be a public accessible file
+| source_pages_numbers | Array    | No       | List of page numbers from pdf file to be added into the publication
+| position             | Integer  | No       | Page number where new pages should be added (by default add pages at the end)
+| extraction_options   | Object   | No       | Define settings to extract hotspots from the PDF file. See [extraction options](#extraction-options) for allowed fields
+
+### Response codes
+
+This endpoint returns the `200` response code.
+
+## Replace a publication page
+
+Replace a page on specific publication
+
+### HTTP Request
+
+`PUT https://api.publitas.com/v2/groups/<Group ID>/publications/<Publication ID>/pages/<Page Number>`
+
+### URL Parameters
+
+| Parameter        | Description
+| ---------        | --------------------------
+| Group ID         | The ID of a specific group
+| Publication ID   | The ID of a specific publication
+| Page Number      | The page number of page to be replaced
+
+
+```shell
+curl --location --request PUT 'https://api.publitas.com/v2/groups/<Group ID>/publications/<Publication ID>/pages/<Page Number>' \
+--header 'Authorization: ApiKey <api_key>' \
+--header 'Content-Type: application/json' \
+--data '{
+  "page": {
+    "source_url": "https://some/file.pdf"
+  }
+}'
+```
+
+### Request body parameters
+
+The following fields need to be sent within a page scope (see right for an example.)
+
+| Name                 | Type     | Required | Description
+| -------------------- | -------- | -------- | -----------
+| source_url           | String   | Yes      | URL where the PDF file resides. HTTP and HTTPS are accepted, and it needs to be a public accessible file
+| source_pages_number  | Integer  | No       | PDF page number to be used on replace (by default page number 1 is used)
+| extraction_options   | Object   | No       | Define settings to extract hotspots from the PDF file. See [extraction options](#extraction-options) for allowed fields
+| remove_hotspots      | Boolean  | No       | If true remove current hotspots on page to be replaced
+
+### Response codes
+
+This endpoint returns the `200` response code.
+
+## Remove a publication page
+
+### HTTP Request
+
+`DELETE https://api.publitas.com/v2/groups/<Group ID>/publications/<Publication ID>/pages/<Page Number>`
+
+```shell
+curl -H "Authorization: ApiKey <api_key>" -X DELETE "https://api.publitas.com/v2/groups/1/publications/222"
+```
+
+### URL Parameters
+
+| Parameter        | Description
+| ---------        | --------------------------
+| Group ID         | The ID of a specific group
+| Publication ID   | The ID of a specific publication
+| Page Number      | The page number to be removed
+
+
+### Response codes
+
+This endpoint returns the `200` response code.
+
+# Product Feeds
+
+## List recent product feeds
+
+List last 5 product feed imports
```

Comment:
`List the last 5 product feed imports` (imperative) or `Lists the last 5 product feed imports` (what it does).